app/javascript/angular/code/controllers/_sessions_list_ctrl.js

```diff
@@ -43,44 +43,17 @@ export const SessionsListCtrl = (
     );
   };
 
-  $scope.sessionFetchCondition = function() {
-    return {
-      id: sensors.selectedId(),
-      params: params.getWithout("data", "heat")
-    };
-  };
-
-  $scope.updateSessionsPage = function() {
-    $scope.page++;
-  };
-
-  $scope.$watch(
-    "page",
-    () => {
-      console.log("watch - page");
-      sessions.fetch($scope.page);
-    },
-    true
-  );
-
   $scope.$watch(
     "params.get('map')",
     ({ hasChangedProgrammatically }) => {
       console.log("watch - params.get('map')");
       if (sessions.hasSelectedSessions()) return;
-      if (!hasChangedProgrammatically) $scope.page = 0;
-      sessions.fetch($scope.page);
-    },
-    true
-  );
-
-  $scope.$watch(
-    "sessionFetchCondition()",
-    (newValue, oldValue) => {
-      console.log("watch - sessionFetchCondition()", newValue, oldValue);
-      if (sessions.hasSelectedSessions()) return;
-      $scope.page = 0;
-      sessions.fetch($scope.page);
```

Comment:
I removed this watch cause it was impossible to track what's going on. There were two watches:
1. on line 66 - which reloads sessions when map moves
2. this one - which reloads sessions when something in the params data changes (so when some of the filters change)
The problem is that when the page is open for the first time both watches where triggered and probable sessions where fetched twice - anyway it was impossible to track what's going on.

I think this watch is not necessary, since we are calling sessions.fetch directly from mobile_sessions_map_crl and fixed_sessions_map_crl when something changes with the filters.

But I'm leaving this longish explanation here, cause it's possible that I missed some use case for this watch - so if you ever notice that the session list is not reloaded when it should - this might be one of the reasons.